pom.xml

```diff
@@ -593,9 +593,14 @@
               <exclude>org.apache.parquet.hadoop.thrift.TBaseWriteSupport#setThriftClass(org.apache.parquet.conf.ParquetConfiguration,java.lang.Class)</exclude>
               <exclude>org.apache.parquet.proto.ProtoParquetReader#builder(org.apache.hadoop.fs.Path,boolean)</exclude>
               <exclude>org.apache.parquet.proto.ProtoParquetReader#builder(org.apache.parquet.io.InputFile,boolean)</exclude>
-
               <!-- Due to the removal of deprecated methods -->
               <exclude>org.apache.parquet.arrow.schema.SchemaMapping</exclude>
+              <!-- Some new methods are added to interface to support SizeStatistics-->
```

Comment:
This is for API breaking changes?

Reply:
Yes, japicmp plugin deems adding new method to public interface is a breaking change.

Reply:
We can avoid exclusions by adding default implementations to the interfaces. We can even add throwing exceptions to enforce implementation. Exclusions tend to remain after a release and makes it hard to catch potential issues in the excluded parts later.

Reply:
Thanks for the suggestion! Fixed.